### PR TITLE
Added example, some cns module corrections/additions

### PR DESCRIPTION
**Pull request by amjjbonvin**

### CODE REVIEW

File: examples/docking-protein-protein/docking-protein-protein.cfg [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# Parameters for each stage are defined below, prefer full paths
+#####################################################################
+# WARNING: THE PARAMETERS HERE ARE ILLUSTRATIVE
+#  THE WORKFLOW IS WORK-IN-PROGRESS
+#####################################################################
+[topoaa]
+autohis = true
+
+[rigidbody]
+ambig_fname = './data/e2a-hpr_air.tbl'
```
Comment (joaomcteixeira) [inline examples/docking-protein-protein/docking-protein-protein.cfg:25]: there's no need for the `./` but it also doesn't harm.

**Review by joaomcteixeira — APPROVED** [review]
Can't find anything wrong.

**Comment by joaomcteixeira** [comment]
I am receiving an error when running the example:

```
Traceback (most recent call last):
  File "/trinity/login/joaomctx/miniconda3/envs/haddock3/bin/haddock3", line 33, in <module>
    sys.exit(load_entry_point('haddock3', 'console_scripts', 'haddock3')())
  File "/trinity/login/joaomctx/haddock3/src/haddock/clis/cli.py", line 67, in maincli
    cli(ap, main)
  File "/trinity/login/joaomctx/haddock3/src/haddock/clis/cli.py", line 62, in cli
    main(**vars(cmd))
  File "/trinity/login/joaomctx/haddock3/src/haddock/clis/cli.py", line 143, in main
    workflow.run()
  File "/trinity/login/joaomctx/haddock3/src/haddock/libs/libworkflow.py", line 25, in run
    step.execute()
  File "/trinity/login/joaomctx/haddock3/src/haddock/libs/libworkflow.py", line 104, in execute
    module.run(**self.config)
  File "/trinity/login/joaomctx/haddock3/src/haddock/modules/sampling/rigidbody/__init__.py", line 154, in run
    haddock_score = HaddockModel(model).calc_haddock_score(**weights)
  File "/trinity/login/joaomctx/haddock3/src/haddock/gear/haddockmodel.py", line 8, in __init__
    self.energies = self._load_energies(pdb_f)
  File "/trinity/login/joaomctx/haddock3/src/haddock/gear/haddockmodel.py", line 13, in _load_energies
    with open(pdb_f) as fh:
FileNotFoundError: [Errno 2] No such file or directory: '/trinity/login/joaomctx/haddock3/examples/docking-protein-protein/run1/01_rigidbody/rigidbody_1.pdb'
```

**Comment by amjjbonvin** [comment]
The error is normal - something goes wrong because of the broken ensemble treatment. But @rvhonorato needs the data to debug things

**Pull request merged**